### PR TITLE
add support for max-ssrcs-per-user config

### PR DESCRIPTION
**Pull request by luocongqiu**

### CODE REVIEW

File: jicofo/rootfs/defaults/jicofo.conf [hunk 1]
```diff
@@ -24,6 +24,8 @@
 {{ $XMPP_RECORDER_DOMAIN := .Env.XMPP_RECORDER_DOMAIN | default "recorder.meet.jitsi" -}}
 {{ $XMPP_PORT := .Env.XMPP_PORT | default "5222" -}}
 {{ $XMPP_SERVER := .Env.XMPP_SERVER | default "xmpp.meet.jitsi" -}}
+{{ $MAX-SSRCS-PER-USER := .Env.MAX-SSRCS-PER-USER | default "20" -}}
```
Comment (saghul) [inline jicofo/rootfs/defaults/jicofo.conf:27]: Please use the underscore _ instead of the hyphen - for the variable name.

Comment (saghul) [reply jicofo/rootfs/defaults/jicofo.conf:27]: Is 20 the JVB default?

File: jicofo/rootfs/defaults/jicofo.conf [hunk 2]
```diff
@@ -24,6 +24,8 @@
...
+{{ $MAX-SSRC-GROUPS-PER-USER := .Env.MAX-SSRC-GROUPS-PER-USER | default $MAX-SSRCS-PER-USER -}}
```
Comment (saghul) [inline jicofo/rootfs/defaults/jicofo.conf:28]: What's the default in the JVB?

**Review by saghul — CHANGES_REQUESTED** [review]

File: jicofo/rootfs/defaults/jicofo.conf (hunk 2 above)
Comment (luocongqiu) [reply jicofo/rootfs/defaults/jicofo.conf]: @saghul    default value is 20 in jicofo

image

**Review by saghul — APPROVED** [review]

**Review by saghul — APPROVED** [review]

**Pull request merged**